Resources/Prototypes/_Arc/Admin.yml

```
@@ -810,18 +810,18 @@
   parent: BasePlushie
   id: PlushieadminFBI
   name: FBI plushie
-  description: A stuffed toy that resembles an ancient earth government facility known as the "FBI".
+  description: A stuffed toy that resembles the insignia of an ancient earth government facility known as the "FBI".
   components:
   - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)